data-models/motion/schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/motion/schema.json",
+    "title": "Motion",
+    "description": "Motion measurements of physical objects",
+    "type": "object",
+    "$defs": {
+        "acceleration": {
+            "G": {
+                "measurementType": "akenza/motion/acceleration/G",
+                "title": "Acceleration",
+                "description": "Acceleration in G",
+                "type": "number",
+                "unit": "G"
+            },
+            "mG": {
+                "measurementType": "akenza/motion/acceleration/mG",
+                "title": "Acceleration",
+                "description": "Acceleration in mG",
+                "type": "number",
+                "unit": "mG"
+            },
+            "m/s^2": {
+                "measurementType": "akenza/motion/acceleration/m/s^2",
+                "title": "Acceleration",
+                "description": "Acceleration in m/s^2",
+                "type": "number",
+                "unit": "m/s^2"
+            }
+        },
+        "speed": {
+            "km/h": {
+                "measurementType": "akenza/motion/speed/km/h",
+                "title": "Speed",
+                "description": "Speed in km/h",
+                "type": "number",
+                "unit": "km/h"
+            },
+            "m/s": {
```

Comment:
@simonrieser this will likely not work properly because the value needs to be URL encoded in order to load the schema. Therefore I suggest, either replacing special symbols or writing them in text

`mPerS` or `ms`

Reply:
@mariusgiger true, subtype should not include any special chars. We e.g. use "mcgm3" for ug/m3. 

Can you fix these 3 instances?
acceleration: ms2
speed: ms, kmh

🙏